mut/stage.py

```diff
@@ -639,33 +609,32 @@ def main() -> None:
         logger.error('Error compiling regular expression: %s', str(err))
         sys.exit(1)
 
-    # --destage requires that we create a Staging context
-    if mode_destage:
-        mode_stage = True
-
     if mode_stage:
         staging = Staging(config)
     elif mode_deploy:
         staging = DeployStaging(config)
 
-    staging.s3.dry_run = dry_run
-
-    if mode_destage:
-        staging.purge()
-        return
-
-    staging.s3.dry_run = dry_run
-
     try:
         do_stage(root, staging)
-    except boto.exception.S3ResponseError as err:
+        staging.changes.print()
+
+        if (not dry_run) and input('Commit? (y/n): ') == 'y':
```

Comment:
Any reason we have to confirm twice on a deploy (not that it's a big deal)? Like we still have to "Press any key to perform the previous" and now commit after doing that

Reply:
I want to phase out that "Press any key..." hack in the makefiles